### PR TITLE
updated course/chapter cards

### PR DESCRIPTION
**Pull request by chrisdugne**
refreshed course/chapter cards to look like 
https://app.zeplin.io/project/5c6d8091c49d450d828bb204/screen/5ce266493355f82ab27553bf

### CODE REVIEW

File: packages/@coorpacademy-components/src/molecule/card/index.js [hunk 1]
```diff
@@ -9,25 +9,39 @@ import {
...
 import Favorite from './favorite';
 import Notification from './notification';
 import style from './style.css';
 
-const viewStyle = {
-  grid_course: classnames(style.grid, style.course),
-  list_course: classnames(style.list, style.course),
-  grid_chapter: classnames(style.grid, style.chapter),
-  list_chapter: classnames(style.list, style.chapter)
+const createSpecificIcon = (whiteColor, adaptiv, type) => {
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/molecule/card/index.js]: `const resourceIcon`

Comment (chrisdugne) [reply packages/@coorpacademy-components/src/molecule/card/index.js]: ok `contentTypeIcon`

**Comment by chrisdugne** [comment]
[ETQ marketing j'aimerais un mode démo marketé](https://trello.com/c/f75JyLhq/1236-etq-marketing-jaimerais-un-mode-d%C3%A9mo-market%C3%A9)

**Comment by chrisdugne** [comment]
[ETQ Coorp j'aimerais unifier le design des cartes microlearning entre le web et le mobile](https://trello.com/c/ImPP4NSl/1303-etq-coorp-jaimerais-unifier-le-design-des-cartes-microlearning-entre-le-web-et-le-mobile)

**Pull request merged**